components/hab/install.sh

```diff
@@ -202,7 +230,37 @@ validate_target() {
   fi
 }
 
-download_archive() {
+download_packages_chef_io_archive() {
+  need_cmd mv
+
+  _version="${version:-latest}"
+
+  if [ "$_version" == "latest" ]; then
+    url="${PCIO_ROOT}/${channel}/latest/habitat/hab-${target}.${ext}"
+  else 
+    url="${PCIO_ROOT}/files/habitat/${version}/hab-${target}.${ext}"
+  fi
+  
+  dl_file "${url}" "${workdir}/hab-${version}.${ext}"
+  dl_file "${url}.sha256sum" "${workdir}/hab-${version}.${ext}.sha256sum"
+
+  archive="hab-${target}.${ext}"
+  sha_file="hab-${target}.${ext}.sha256sum"
+
+  mv -v "${workdir}/hab-${version}.${ext}" "${archive}"
+  mv -v "${workdir}/hab-${version}.${ext}.sha256sum" "${sha_file}"
+  
+  if command -v gpg >/dev/null; then
+    info "GnuPG tooling found, downloading signiatures"
+    sha_sig_file="${archive}.sha256sum.asc"
+    key_file="${workdir}/habitat.asc"
+
+    dl_file "${url}.sha256sum.asc" "${sha_sig_file}"
+    dl_file "${url}.asc" "${key_file}" 
```

Comment:
This is currently not the correct file.

Reply:
This is something I needed to chat with you about :smile:  It should be the packages.chef.io equivalent of  `https://bintray.com/user/downloadSubjectPublicKey?username=habitat`